citesphere/src/main/java/edu/asu/diging/citesphere/core/authority/impl/ViafAuthorityImporter.java

```diff
@@ -50,6 +48,9 @@ public class ViafAuthorityImporter extends BaseAuthorityImporter {
     @Value("${_viaf_url}")
     private String viafURL;
 
+    @Value("${_importer_name_authority.importer.viaf}")
+    private String ID;
```

Comment:
it should stay the way it was. Each importer has a hard-coded id, that id should be mapped to a name that's in the properties file.